src/Layout/Footer/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import styles from './styles.module.css'
+import { FaFacebookF, FaEnvelope } from 'react-icons/fa'
+import { AiFillInstagram, AiOutlineTwitter } from 'react-icons/ai'
+import { linkPaths } from '../../LinksPaths'
+import { Link } from 'react-router-dom'
+import { Button } from '@mui/material'
+
+const contactLinks = [
+  { label: 'contact', path: linkPaths.contact },
+  { label: 'assistance', path: linkPaths.assistance },
+  { label: 'about us', path: linkPaths.aboutUs }
+]
+const socialMediaLinks = [
+  { icon: <FaFacebookF size={'2em'} />, path: linkPaths.facebook },
+  { icon: <AiFillInstagram size={'3em'}/>, path: linkPaths.instagram },
+  { icon: <AiOutlineTwitter size={'3em'}/>, path: linkPaths.twitter },
+  { icon: <FaEnvelope size={'2.5em'}/>, path: linkPaths.mail }
+]
+
+function uuidv4 () {
+  return ([1e7]+-1e3+-4e3+-8e3+-1e11).replace(/[018]/g, c => //eslint-disable-line
+    (c ^ crypto.getRandomValues(new Uint8Array(1))[0] & 15 >> c / 4).toString(16) //eslint-disable-line
+  )
+}
+
+export function Footer ({ children }) {
+  return (
+    <footer className={styles.footer}>
+      { children }
+      <div className={styles.contactBar}>
+        {contactLinks.map(link => (
+          <Link key={uuidv4()} to={link.path}>
```

Comment:
```suggestion
          <Link key={link.path} to={link.path}>
```
How about using `path` property? It is unique already.

Reply:
P.S. If you will ever need to use uuid as key, put it in the array declaration, for example as property of the `link` object. Thanks to that the `key` will not be changing with every rerender. More reading here https://stackoverflow.com/questions/52705335/will-using-uuid-to-as-list-keys-cause-unnecessary-re-renders-in-react

Reply:
an here https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/